main.py

```diff
@@ -1,10 +1,48 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as json_file:
+        players = json.load(json_file)
+
+    for key, player in players.items():
+        if player["race"]["skills"]:
+            for skill in player["race"]["skills"]:
+                Skill.objects.get_or_create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=Race.objects.get_or_create(
+                        name=player["race"]["name"],
+                        description=player["race"]["description"],
+                    )[0],
+                )
+
+    for key, player in players.items():
+
+        Player.objects.get_or_create(
+            nickname=key,
+            email=player["email"] if player["email"] else None,
+            bio=player["bio"] if player["bio"] else None,
```

Comment:
This code duplicates the logic for checking if 'player["email"]' and 'player["bio"]' are present. According to the checklist, avoid duplication by using a more concise expression.